platform/config/config-model-api/src/main/java/org/codice/ddf/config/model/CswFederationProfileConfig.java

```diff
@@ -13,9 +13,7 @@
  */
 package org.codice.ddf.config.model;
 
-import org.codice.ddf.config.ConfigType;
-
-@ConfigType
+// @ConfigType
 public interface CswFederationProfileConfig extends SourceConfig {
 
   @Override
```

Comment:
This method should be moved up to the SourceConfig class to return SourceConfig.class and let's  remove the commented out @ConfigType

Reply:
fixed